### PR TITLE
Pptp 1448

### PR DESCRIPTION
**Pull request by ajchilts**
### Description of Work carried through

Brief description of what/why/how.

#### Check list 
 - [x] `./precheck` was executed (Integration/Component/Unit tests)
 - [ ] `sbt scalafmt test:scalafmt` was executed 
 - [ ] Required Environment Config has been amended/added
 - [ ] Links to dependencies have been included (BE/FE work)
 - [ ] User Acceptance Tests (UAT) were run locally and have passed.
 - [ ] No Accessibility errors/warnings reported by Wave

### CODE REVIEW

File: app/controllers/SubmittedReturnsController.scala [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+  def onPageLoad: Action[AnyContent] =
+    identify.async {
+
+      implicit request =>
+        val pptReference =
+          request.enrolmentId.getOrElse(throw new IllegalStateException("no enrolmentId"))
+
+        obligationsConnector.getFulfilled(pptReference).flatMap { taxReturnObligations =>
+          Future.successful(Ok(view(taxReturnObligations)))
```
Comment (liampan) [inline app/controllers/SubmittedReturnsController.scala:45]:
```suggestion
        obligationsConnector.getFulfilled(pptReference).map { taxReturnObligations =>
          Ok(view(taxReturnObligations))
```

File: app/views/AmendConfirmation.scala.html [hunk 2]
```diff
@@ -38,5 +38,9 @@
 <p class="govuk-body">The amount of tax you owe may have changed.</p>
```
Comment (liampan) [inline app/views/AmendConfirmation.scala.html:38]: these peices of content should be in messages, happy for future todo

File: app/views/AmendImportedPlasticPackagingView.scala.html [hunk 3]
```diff
@@ -36,7 +36,7 @@
     <li>an integral part of the goods</li>
     <li>reused for the presentation of goods</li>
```
Comment (liampan) [inline app/views/AmendImportedPlasticPackagingView.scala.html:37]: and here

File: conf/messages.en [hunk 4]
```diff
@@ -262,3 +268,18 @@ account.homePage.card.registration.details.2.body.partnership = View and change
 
 account.homePage.card.deregister.link = Deregister from Plastic Packaging Tax
 account.homePage.card.deregister.body = Cancel your Plastic Packaging Tax registration.
+
+viewReturnSummary.summary.field.1 = Tax liability for this period
+viewReturnSummary.summary.field.2 = Return submitted
+viewReturnSummary.summary.field.3 = Payment due
+viewReturnSummary.summary.field.4 = Return reference number
```
Comment (liampan) [inline conf/messages.en:275]: are these right?

**Review by liampan — APPROVED** [review]

File: conf/messages.en [hunk 5]
```diff
@@ -262,3 +268,18 @@ account.homePage.card.registration.details.2.body.partnership = View and change
 
 account.homePage.card.deregister.link = Deregister from Plastic Packaging Tax
 account.homePage.card.deregister.body = Cancel your Plastic Packaging Tax registration.
+
```
Comment (mauro-fanton) [inline conf/messages.en:271]: are we putting place-hold for the welsh translation in the message.cy for these?

File: conf/messages.en (hunk 4 above)
Comment (mauro-fanton) [reply conf/messages.en:275]: yes, the above seem are not used

**Pull request merged**